docs/ossm/cert-manager/README.md

```diff
@@ -0,0 +1,253 @@
+
+    ```sh
+    oc apply -f - <<EOF
+    apiVersion: sailoperator.io/v1alpha1
+    kind: Istio
+    metadata:
+      name: default
+    spec:
+      version: v1.23.0
+      namespace: istio-system
+      values:
+        global:
+          caAddress: cert-manager-istio-csr.cert-manager.svc:443
+        pilot:
+          env:
+            ENABLE_CA_SERVER: "false"
+          volumeMounts:
+            - mountPath: /tmp/var/run/secrets/istiod/tls
+              name: istio-csr-dns-cert
+              readOnly: true
+    EOF
+    ```
+
+6.  Verification
+
+    Use the sample httpbin service and sleep app to check traffic between the workloads is possible and check the workload certificate of the proxy to verify that the cert-manager tool is installed correctly.
+
+    a. Create the `sample` namespace:
+
+    ```sh
+    oc new-project sample
+    ```
+
+    b. Find your active `IstioRevision`:
+
+    ```sh
+    oc get istiorevisions
+    ```
+
+    c. Add the injection label for your active revision to the `sample` namespace:
+
+    ```sh
+    oc label namespace sample istio.io/rev=<your-active-revision-name> --overwrite=true
+    ```
+
+    d. Deploy the HTTP and sleep apps:
+
+    ```sh
+    oc apply -n sample -f https://raw.githubusercontent.com/istio/istio/refs/heads/master/samples/httpbin/httpbin.yaml
+    oc apply -n sample -f https://raw.githubusercontent.com/istio/istio/refs/heads/master/samples/sleep/sleep.yaml
+    oc rollout status deployment httpbin sleep
+    ```
+
+    e. Verify that sleep can access the httpbin service:
+
+    ```sh
+    oc exec "$(oc get pod -l app=sleep -n sample \
+      -o jsonpath={.items..metadata.name})" -c sleep -n sample -- \
+      curl http://httpbin.sample:8000/ip -s -o /dev/null \
+      -w "%{http_code}\n"
+    ```
+
+    Example output
+
+    ```sh
+    200
+    ```
+
+    f. Verify `httpbin` workload certificate matches what is expected:
+
+    ```sh
+    istioctl proxy-config secret -n sample $(oc get pods -n sample -o jsonpath='{.items..metadata.name}' --selector app=httpbin) -o json | jq -r '.dynamicActiveSecrets[0].secret.tlsCertificate.certificateChain.inlineBytes' | base64 --decode | openssl x509 -text -noout
+    ```
+
+    Example output
+
+    ```sh
+    ...
+    Issuer: O = cert-manager + O = cluster.local, CN = istio-ca
+    ...
+    X509v3 Subject Alternative Name:
+      URI:spiffe://cluster.local/ns/sample/sa/httpbin
+    ```
+
+### `RevisionBased` Upgrades
+
+This section only applies to `RevisionBased` deployments.
+
+Because istio-csr requires you to pass all revisions, each time you upgrade your `RevsionBased` controlplane you will need to **first** update your istio-csr deployment with the new revision before you update your `Istio.spec.version`. For example, before upgrading your controlplane from `v1.23.0 --> v1.23.1`, you need to first update your istio-csr deployment with the new revision:
```

Comment:
```suggestion
Because istio-csr requires you to pass all revisions, each time you upgrade your `RevisionBased` controlplane you will need to **first** update your istio-csr deployment with the new revision before you update your `Istio.spec.version`. For example, before upgrading your controlplane from `v1.23.0 --> v1.23.1`, you need to first update your istio-csr deployment with the new revision:
```